common/static/common/js/karma.common.conf.js

```diff
@@ -269,8 +271,15 @@ function getBaseConfig(config, useRequireJs) {
     };
 
     var hostname = 'localhost';
+    var port = 9876;
     if (process.env.hasOwnProperty('BOK_CHOY_HOSTNAME')) {
         hostname = process.env.BOK_CHOY_HOSTNAME;
+        if (hostname === 'edx.devstack.lms') {
+            port = 19876;
```

Comment:
Is there documentation that needs updating with the various ports that Docker uses?